include/examples/simple_io_behavior.h

```diff
@@ -0,0 +1,146 @@
+/**
+ * @file simple_io_behavior.h
+ * @author Ethan J. Musser (emusser@seas.upenn.edu)
+ * @brief Defines a simple behavior class which provides position control for a
+ * single joint in a `SimpleRobot`, accepting gain commands via LCM input, and
+ * logging joint position via LCM output.
+ * @date 8/10/22
+ * 
+ * @copyright (c) Copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include <vector>
+
+#include "PDGains.hpp"  // PDGains
+#include "MotorLog.hpp"  // MotorLog
+#include "real_time_tools/timer.hpp"  // real_time_tools::Timer
+
+#include "kodlab_mjbots_sdk/io_behavior.h"  // kodlab::IOBehavior
+#include "kodlab_mjbots_sdk/log.h"  // logging macros
+#include "examples/simple_robot.h"  // SimpleRobot
+
+/**
+ * @brief Simple behavior that runs a positional PD control loop on the first
+ * joint in a SimpleRobot with gains adjustable via LCM input and motor state
+ * logged via LCM output
+ */
+class SimpleIOBehavior : public kodlab::IOBehavior<SimpleRobot,
```

Comment:
SimpleControlIOBehavior or something
Again just so people know what is being run apriori

Reply:
Changed in f4b04a6f22a7cb91e3c61c238b5b25a3686f1eef